css/themes/default/jquery.mobile.theme.css

```diff
@@ -611,3 +613,22 @@ button[disabled],
 	outline-width: 1px;
 	outline-style: inherit;
 }
+
+.ui-accordion-header {
```

Comment:
These should use existing styles and account for theme variables. The header should probably share with the `ui-button` class the styles should share. This needs to be in both the theme a / b sections as well